### PR TITLE
Add a marker on my submissions list which are shown on the leaderboard

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- Added a green color check icon at the start of team name

- Also shown the purpose of this icon over the list by displaying the icon and and written `Shown on leaderboard` in front of it

- Link to live demo: http://pr-164-evalai.surge.sh 

**<p align="center">Screenshot</p>**

Screenshot from 2019-07-07 18-19-30

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@Sanji515 The travis build is failing.

**Comment by Sanji515** [comment]
@RishabhJain2018 yeah because this PR is build on top of all my previous PR, so first those need to be merge

**Comment by RishabhJain2018** [comment]
I see..thanks for letting me know. I'll get those merge soon.

**Comment by Sanji515** [comment]
@lunayach @Shekharrajak I've resolved the conflicts here, please review.

File: src/app/components/challenge/challengesubmissions/challengesubmissions.component.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
           </div>
         </div>
       </div>
+      <div *ngIf="submissions.length > 0" class="display-flex pad-top"><i class="fa fa-check-circle submitted marker"></i><span class="show-on-leaderboard">Shown on leaderboard</span></div>
```
Comment (lunayach) [inline src/app/components/challenge/challengesubmissions/challengesubmissions.component.html]:
```suggestion
      <div *ngIf="submissions.length > 0" class="display-flex pad-top"><i
        class="fa fa-check-circle submitted marker"></i><span class="show-on-leaderboard">Shown on leaderboard</span>
      </div>
    </div>
```

Comment (lunayach) [reply src/app/components/challenge/challengesubmissions/challengesubmissions.component.html]: That is, not everything in one line.

**Comment by RishabhJain2018** [comment]
Screenshot 2019-07-20 at 3 25 30 PM
@Sanji515 Please make this color consistent with the orange in current version.

**Comment by Sanji515** [comment]
> Please make this color consistent with the orange in current version.

@RishabhJain2018 Actually this will get resolved when the leaderboard PR will be merge.

@lunayach Made the changes, please review

**Review by lunayach — APPROVED** [review]

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**